doc/connectivity/bluetooth/api/audio/shell/cap.rst

```diff
@@ -451,3 +451,7 @@ Starting a broadcast reception
    BASS discover done with 1 recv states
    uart:~$ cap_commander broadcast_reception_start <device B> 0 4
    Starting broadcast reception on 1 connection(s)
+   Broadcast reception start completed
+   uart:~$ cap_commander broadcast_reception_stop <src_id>
```

Comment:
```suggestion
   uart:~$ cap_commander broadcast_reception_stop 0
```

Right?

Also, both `broadcast_reception_stop` and `broadcast_reception_start` are not documented in table above: https://github.com/zephyrproject-rtos/zephyr/pull/76322/files#diff-ead796b94559421f9e06247dca73bfe10daf262080a8c981f47b62ab24839ebcR257-R268